underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystemFactory.java

```diff
@@ -101,6 +101,12 @@ private boolean addAndCheckSwiftCredentials() {
       Configuration.set(passwordKeyConf, System.getProperty(passwordKeyConf));
     }
 
+    String swiftSimulationConf = Constants.SWIFT_SIMULATION;
+    if (System.getProperty(swiftSimulationConf) != null
+        && !Configuration.containsKey(swiftSimulationConf)) {
```

Comment:
In that case, do we want to check whether `configuration.get(swiftSimulationConf) == null` as well?